lib/std/c.zig

```diff
@@ -9351,6 +9351,7 @@ pub extern "c" fn setlogmask(maskpri: c_int) c_int;
 pub extern "c" fn if_nametoindex([*:0]const u8) c_int;
 
 pub extern "c" fn getpid() pid_t;
+pub extern "c" fn getppid() pid_t;
```

Comment:
Note that this doesn't exist on Windows.

It's "fine" in that it's no worse than many of the other declarations immediately surrounding it. Just noting that at some point we need to go through and change these declarations to be in line with the principles laid out in #20679. I don't think this necessarily has to block merging this PR, since that conversion work will likely be done in a batched fashion anyway.